### PR TITLE
Switch to Cloud 8 development

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1180,10 +1180,10 @@ function onadmin_set_source_variables
         GM7|GM7+up|GMC*|M?)
```
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: Don't we need to keep GM7 and GM7+up for SOC7?

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -1180,10 +1180,10 @@ function onadmin_set_source_variables
...
             cs=${cloudsource/#M/Milestone}
             [[ $cs =~ GM7 ]] && cs=GM
-            CLOUDISOPATH=${want_cloud7_iso_path:="/install/SLE-12-SP2-Cloud7-$cs/"}
-            CLOUDISONAME=${want_cloud7_iso:="SUSE-OPENSTACK-CLOUD-7-${arch}*1.iso"}
-            CLOUDTESTISONAME="CLOUD-7-TESTING-$arch*DVD1.iso"
-            CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-7-official"
+            CLOUDISOPATH=${want_cloud7_iso_path:="/install/SLE-12-SP3-Cloud8-$cs/"}
```
Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Shouldn't we rather introduce another label such as `GM8`?

Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: This makes sense for `GMC*|M?` , but why for `GM7|GM7+up` ?

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: yeah, probably.

**Pull request merged**